content/en/service_catalog/guides/troubleshooting.md

```diff
@@ -16,10 +16,6 @@ If services that you know are instrumented for APM are not appearing in the Serv
 
 {{< img src="tracing/service_catalog/svc_cat_troubleshooting_1.png" alt="Hover message indicating that no performance data has been reported in the past hour" >}}
 
-## Extra services are discovered as an artifact of APM instrumentation 
```

Comment:
It's still true that you can remove these along with opting into the beta. We just aren't making this a requirement of the beta. I don't think we need to remove this.

Reply:
Even though this is true, as we already have close to 200 candidates for the beta, I don't think we need to advertise the beta anymore in other places in the docs (yet) - at least until we are in a more stable state where we can enable the feature for larger batches of customers.